subsys/bluetooth/Kconfig

```diff
@@ -128,6 +128,7 @@ config BT_MAX_CONN
 	int "Maximum number of simultaneous connections"
 	depends on BT_CONN
 	range 1 64
+	default 2 if BT_MESH_GATT_CLIENT
```

Comment:
`BT_MAX_CONN` and `BT_RX_STACK_SIZE` stay the same if GATT Client is enabled through menuconfig. Probably `range` could fix that, but not sure if it is the right way to do.